pkg/local_object_storage/metabase/version_test.go

```diff
@@ -307,16 +308,20 @@ func TestMigrate2to3(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	// inhumeV2 stores data in the old format, but new DB has current version by default, force old version.
 	err = db.boltDB.Update(func(tx *bbolt.Tx) error {
-		err = updateVersion(tx, 2)
-		if err != nil {
-			return err
-		}
-
-		return migrateFrom2Version(db, tx)
+		return updateVersion(tx, 2)
 	})
 	require.NoError(t, err)
 
+	db.mode = mode.DegradedReadOnly // Force reload.
+	ok, err := db.Reload(WithPath(db.info.Path), WithEpochState(epochState{}))
+	require.True(t, ok)
+	require.NoError(t, err)
```

Comment:
better to check before `ok` to know what exactly failed if anything

Reply:
Yes, that's the way I got to the right parameters, but it somehow changed before hitting GH. Fixed.